Assets/Script/Menu/ProfileList/ProfileView.cs

```diff
@@ -36,6 +35,7 @@ public class ProfileView : NavigatableBehaviour
         private ProfileListMenu _profileListMenu;
         private ProfileSidebar _profileSidebar;
         private YargProfile _profile;
+        public YargProfile Profile => _profile;
```

Comment:
This can be converted to a property with a private setter. If this happens, it should also be moved above the two private fields there, but below the serialized ones.
```suggestion
        public YargProfile Profile { get; private set; }
```